tests/integration-tests/tests/performance_tests/test_openfoam.py

```diff
@@ -65,21 +61,24 @@ def test_openfoam(
             additional_files=[str(test_datadir / "openfoam.slurm.sh")],
             timeout=OPENFOAM_JOB_TIMEOUT,
         )
-        # pid = read_remote_file(remote_command_executor, '/tmp/openfoam.pid')
-        # logging.info(f"Waiting for OpenFOAM job to complete with pid = {pid}")
-        # wait_process_completion(remote_command_executor, pid)
+
         perf_test_result = remote_command_executor.run_remote_script(
             (str(test_datadir / "openfoam.results.sh")), hide=False
         )
         output = perf_test_result.stdout.strip()
         elapsed_time = output.split("\n")[-1].strip()
-        logging.info(f"The elapsed time for {node} nodes is {elapsed_time}")
+        baseline_value = BASELINE_CLUSTER_SIZE_ELAPSED_SECONDS[node]
+        logging.info(f"The elapsed time for {node} nodes is {elapsed_time} seconds")
         percentage_difference = perf_test_difference(int(elapsed_time), node)
-        logging.info(
-            f"Percentage difference for cluster size {node} between observed elapsed time "
-            f"({elapsed_time}) and baseline ({BASELINE_CLUSTER_SIZE_ELAPSED_SECONDS[node]}):"
-            f" {percentage_difference}"
-        )
+        outcome = "degradation" if percentage_difference > 0 else "improvement"
+        logging.info(f"Nodes: {node}, Baseline: {baseline_value} seconds, Observed: {elapsed_time} seconds, "
+                     f"Percentage difference: {percentage_difference}%, Outcome: {outcome}")
```

Comment:
An additional useful info is whether or not the degradation is within the tolerance level. We can add in the log line "within tolerance/beyond tolerance"